package/bin/audit_query_helper.py

```diff
@@ -11,6 +11,7 @@
 from dateutil.relativedelta import relativedelta
```

Comment:
Please use `timedelta` instead of `relativedelta` to keep code consistency and reduce dependencies. See other comments.